apps/developer-portal/src/index.html

```diff
@@ -29,20 +29,37 @@
 
         <script src="<%= htmlWebpackPlugin.options.publicPath %>/app-utils.js"></script>
         <script>
-            AppUtils.init({
-                accountAppOrigin: "https://localhost:9000",
-                serverOrigin: "https://localhost:9443",
-                superTenant: "carbon.super",
-                tenantPrefix: "t"
-            });
+            if (window["AppUtils"].getConfig() === null) {
+                AppUtils.init({
+                    accountAppOrigin: "https://localhost:9000",
+                    serverOrigin: "https://localhost:9443",
+                    superTenant: "carbon.super",
+                    tenantPrefix: "t"
+                });
+            }
 
-            var doNotDeleteApplications = ["Developer Portal", "User Portal"];
+            const state = new URL(window.location.href).searchParams.get("state");
+            if (state !== null && state === "checkSession") {
+                // Prompt none response.
+                const code = new URL(window.location.href).searchParams.get("code");
```

Comment:
Change all `const`s to `var`s. Because ES6 syntax's only support in latest browsers.